tests/specs/test/recursive_permissions_pledge/__test__.jsonc

```diff
@@ -4,5 +4,5 @@
   "envs": {
     "RUST_BACKTRACE": "0"
   },
-  "output": "\n============================================================\nDeno has panicked. This is a bug in Deno. Please report this\nat https://github.com/denoland/deno/issues/new.\nIf you can reliably reproduce this panic, include the\nreproduction steps and re-run with the RUST_BACKTRACE=1 env\nvar set and include the backtrace in your report.\n\nPlatform: [WILDCARD]\nVersion: [WILDCARD]\nArgs: [[WILDCARD], \"test\", \"main.js\"]\n\nView stack trace at:\n[WILDCARD]\nthread 'tokio-runtime-worker' panicked at [WILDCARD]testing.rs:[WILDCARD]:\npledge test permissions called before restoring previous pledge\nnote: run with `RUST_BACKTRACE=1` environment variable to display a backtrace\n"
```

Comment:
I added an assertion to prevent people from putting these super long strings here.

This test started failing for me locally because we don't do local builds with the `View stack trace at:` line.

Reply:
Good idea!